frontend/src/components/common/Resource/EditorDialog.tsx

```diff
@@ -134,10 +140,10 @@ export default function EditorDialog(props: EditorDialogProps) {
     }
   }
 
-  function getObjectFromCode(code: string): KubeObjectInterface | null {
+  function getObjectsFromCode(code: string): KubeObjectInterface[] | null {
     try {
-      const codeObj = yaml.load(code);
-      return codeObj as KubeObjectInterface;
+      const codeObjs = yaml.loadAll(code);
+      return codeObjs as KubeObjectInterface[];
```

Comment:
We now also support json. So I guess this is why we're having conflicts.

Reply:
Fixed